pkg/actions/cluster/get.go

```diff
@@ -0,0 +1,139 @@
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	"github.com/kris-nova/logger"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/eks"
+)
+
+const (
+	eksctlCreatedTrue    api.EKSCTLCreated = "True"
+	eksctlCreatedFalse   api.EKSCTLCreated = "False"
+	eksctlCreatedUnknown api.EKSCTLCreated = "Unknown"
+)
+
+type Description struct {
+	Name   string
+	Region string
+	Owned  api.EKSCTLCreated
+}
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
```

Comment:
So the one below doesn't have this, right? I thought this isn't needed. :D or does it only matter once and then it's used for everything? :thunk:

Reply:
If I remember correctly its 1 `go:generate` per package